packages/cozy-harvest-lib/src/services/utils.js

```diff
@@ -47,6 +47,31 @@ export const fetchAccountsForCipherId = async (cozyClient, cipherId) => {
   return accounts
 }
 
+export const fetchKonnectorFromAccount = async (cozyClient, account) => {
+  const konnectorDoctype = 'io.cozy.konnectors'
+  const slug = account.account_type
+  const konnector = await cozyClient.query(
+    cozyClient.get(konnectorDoctype, slug)
+  )
+  return konnector.data
+    ? {
+        _id: konnector.data._id,
+        _type: konnector.data.type,
+        ...konnector.data.attributes
+      }
+    : null
+}
+
+export const fetchTriggersFromAccount = async (cozyClient, account) => {
+  const triggers = await cozyClient.queryAll(
+    cozyClient.find('io.cozy.triggers').where({
+      worker: 'konnector',
+      'message.account': account._id
+    })
+  )
```

Comment:
I wonder if we should not expose query definition creation utilities in cozy client.

```
import { queryTriggersFromAccount } from 'cozy-client'

const triggers = await cozyClient.queryAll(queryTriggersFromAccount(account))
```

Reply:
cc @Crash--

Reply:
I think having this helper in cozy-client/src/models/triggers would make sense.

Reply:
I added an issue https://github.com/cozy/cozy-client/issues/789